tests/python/gaia-ui-tests/gaiatest/apps/ftu/app.py

```diff
@@ -392,11 +395,14 @@ def a11y_click_next_to_finish_section(self):
                 *self._section_finish_locator))))
 
     def tap_skip_tour(self):
-        try:
-            self.marionette.find_element(*self._skip_tour_button_locator).tap()
-        except (FrameSendFailureError, NoSuchWindowException):
-            # The frame may close for Marionette but that's expected so we can continue - Bug 1065933
-            pass
+        element = self.marionette.find_element(*self._skip_tour_button_locator)
```

Comment:
We can refactor these 7 lines into a function, and call it here and in `tap_lets_go_button` too